tests/regression/test_slepc.py

```diff
@@ -33,8 +33,8 @@ def test_laplace_physical_ev(parallel=False):
     # This shifts the "1.0" Eigenvalues out of the spectrum
     # of interest (which is around 0.0 in this case).
     vals = np.repeat(1E8, len(bc.nodes))
-    A.setValuesLocalRCV(bc.nodes.reshape(-1, 1),
-                        bc.nodes.reshape(-1, 1),
+    A.setValuesLocalRCV(bc.nodes.reshape(-1, 1).astype('int32'),
+                        bc.nodes.reshape(-1, 1).astype('int32'),
```

Comment:
This is incorrect (should use `pyop2.datatypes.IntType`), but also, we should ensure that the `nodes` that a BC object returns are _already_ of that type.

Reply:
It once somehow complained about automatic conversion from 'int64' to 'int32', but it's gone now, so astype() is just removed.